inference-engine/thirdparty/ocv/opencv_hal_avx.hpp

```diff
@@ -1966,6 +1978,11 @@ static inline v_uint32x8 v_setr_s32(int b0, int b1, int b2, int b3, int b4, int
     return v_uint32x8(_mm256_setr_epi32(b0, b1, b2, b3, b4, b5, b6, b7));
 }
 
+static inline v_uint32x8 v_setPermuteMask()
```

Comment:
You've mentioned in the call that this should be removed (and may be similar auxiliary functions), just left reminder here as was agreed

Reply:
done